flow/flow.go

```diff
@@ -82,6 +82,7 @@ type Flow struct {
 	Ipv6SrcRoutePrefix []byte
 	Ipv6DstRoutePrefix []byte
 	IsMetric           bool
+	AppProtocol        uint32
```

Comment:
This struct can be optimized to reduce memory usage by about 6% if my napkin math is correct.  I only mention it because this seems like a struct we instantiate a lot.  There are a couple gaps in this struct if we consider the 8-byte alignment.
I think you can add `AppProtocol` without increasing memory usage if you insert it immediately after `TcpRetransmit uint32`

Does the ordering of these fields matter to capn proto?  If not we can optimize it further by grouping the `bool` and `uint8` fields.

Reply:
Created https://github.com/kentik/libkflow/issues/21 as the issue to track the further improvement.